src/c76_menuCAL.hpp

```diff
@@ -94,6 +94,12 @@ enum
 // Start off with Polar Alignment higlighted.
 byte calState = HIGHLIGHT_FIRST;
 
+// Variables to accumulate and smooth the accelerometer readings of the digital level
+#if USE_GYRO_LEVEL == 1
+angle_t accumulator;
+float LPFBeta = 0.6f;
```

Comment:
You should get in the habit of using `const` wherever possible: https://isocpp.org/wiki/faq/const-correctness

Reply:
Sure.